<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="INMUCHLP03121" name="com.automatics.packages.testScripts.amazon_assignment" tests="1" failures="1" timestamp="05 May 2020 06:30:05 GMT" time="79.166" errors="0">
  <testcase name="test" time="79.166" classname="com.automatics.packages.testScripts.amazon_assignment">
    <failure type="java.lang.AssertionError" message="Run Time
Error: Run TimeError: java.lang.NullPointerException	at com.automatics.packages.library.Android_Hybrid.screenShot(Android_Hybrid.java:3608)">
      <![CDATA[java.lang.AssertionError: Run Time
Error: Run TimeError: java.lang.NullPointerExceptionat com.automatics.packages.library.Android_Hybrid.screenShot(Android_Hybrid.java:3608)
at org.testng.Assert.fail(Assert.java:96)
at com.automatics.packages.library.Reporter.printError(Reporter.java:200)
at com.automatics.packages.library.Reporter.printError(Reporter.java:175)
at com.automatics.packages.testScripts.amazon_assignment.test(amazon_assignment.java:96)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- com.automatics.packages.testScripts.amazon_assignment -->
